tests/infra-fixtures/test-uuid.rego

```diff
@@ -0,0 +1,5 @@
+package example
+
+policy[data] {
```

Comment:
actually this is now

```rego
policy[data] {
	data := {
		# these two must be equal during a single query, but different from one invocation to another
		"one": uuid.rfc4122("id"),
		"two": uuid.rfc4122("id"),
	}
}
```
because it now has to ensure that both instantiations should be the same value.